readme.md

```diff
@@ -92,6 +123,12 @@ Type: `array`
 
 Keys to include.
 
+##### shortargs
```

Comment:
Naming is my bane :grimacing:. Do any of these seem more expressive: `shortFlags`, `shortForms`, `singleDashAliases`?